frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -244,6 +247,89 @@ export const funnelLogic = kea<funnelLogicType<FunnelLogicProps>>({
                 },
             },
         ],
+        correlations: [
+            { events: [] } as Record<'events', FunnelCorrelation[]>,
+            {
+                loadCorrelations: async (correlationTypes: FunnelCorrelationType[]) => {
+                    console.log('correlationtype: ', correlationTypes)
+                    return {
+                        events: [
+                            {
+                                event: 'Event A',
+                                correlation_type: 'success',
```

Comment:
@hazzadous I changed my mind a bit while working on the UI.

I think it **doesn't make sense** to send different requests depending on the type of info we want (success vs failure), because it will literally get the same info out, and slice the result. This is expensive when our queries are slow.

Rather, send top 10 success with correlation_type='success', and send top 10 failures with correlation_type='failure', and then filtering on the UI is just a frontend operation. From this list of 20, choose what to discard, what not to discard.

Makes sense?

Reply:
Yes makes sense